provisioning/templates-api/common.sh

```diff
@@ -22,9 +22,6 @@ fi
 # Namespace
 kubectl apply -f ./kubernetes/deploy/namespace.yaml
 
-# REMOVE ME: delete old manually deployed resources
-kubectl delete pod,statefulset,configmap,networkpolicy,secret,service,persistentVolumeClaim,persistentVolume --namespace templates-api -l app=templates-api-etcd --ignore-not-found
-
```

Comment:
Odstranil som docasny kod (to sa releasne samostatne, ked sa releasne predchadzajuca cast).